client/extensions/woocommerce/state/ui/shipping/zones/test/selectors.js

```diff
@@ -41,12 +41,42 @@ describe( 'selectors', () => {
 			const state = {
```

Comment:
all the mocks are getting crazy long,  check out what I did here:  https://github.com/Automattic/wp-calypso/pull/14929/files#diff-47a461893e060930ee76d7c81c1c4bc2R24